src/zenml/integrations/kubernetes/flavors/kubernetes_orchestrator_flavor.py

```diff
@@ -134,6 +134,17 @@ def is_schedulable(self) -> bool:
         """
         return True
 
+    @property
+    def supports_client_side_caching(self) -> bool:
+        """Whether the orchestrator supports client side caching.
+
+        Returns:
+            Whether the orchestrator supports client side caching.
+        """
+        # The Kubernetes orchestrator starts step pods from a pipeline pod.
```

Comment:
Yes, if we had more time there would be multiple solutions for this, but all of them take days of effort as far as I can see.
I'm always a little sceptical to have log messages on simple properties. Why would it log something each time a boolean is read? We don't even know who tries to access it. What do you think instead if we add a log if we skip client-side caching for some reason?

Reply:
Maybe I chose the wrong place to post this comment. What I meant was, how about we display this message (in the commented-out section) as a `debug` message only once when a pipeline is executed? Otherwise, I agree with you. Displaying it every time someone accesses it, is a bad idea.